application/frontend/src/pages/BrowseRootCres/browseRootCres.tsx

```diff
@@ -17,30 +16,29 @@ export const BrowseRootCres = () => {
   const { apiUrl } = useEnvironment();
   const [loading, setLoading] = useState<boolean>(false);
   const [display, setDisplay] = useState<Document[]>();
-  const { error, data, refetch } = useQuery<{ data: Document }, string>(
-    'cre',
-    () =>
-      fetch(`${apiUrl}/root_cres`)
-        .then((res) => res.json())
-        .then((resjson) => {
-          setDisplay(resjson.data);
-          return resjson;
-        }),
-    {
-      retry: false,
-      enabled: false,
-      onSettled: () => {
-        setLoading(false);
-      },
-    }
-  );
+  const [error, setError] = useState<string | Object | null>(null);
 
   useEffect(() => {
-    window.scrollTo(0, 0);
     setLoading(true);
-    refetch();
-  }, []);
+    window.scrollTo(0, 0);
 
+    axios
+      .get(`${apiUrl}/root_cres`)
+      .then(function (response) {
+        setError(null);
+        setDisplay(response?.data?.data);
+      })
+      .catch(function (axiosError) {
+        if (axiosError.response.status === 404) {
+          setError('Standard does not exist in the DB, please check your search parameters');
+        } else {
+          setError(axiosError.response);
```

Comment:
Potential risk of exposing backend information

Reply:
this information is already sent to the client tho, the backend should take care not to send this kind of info